### PR TITLE
N29: Release

### PR DESCRIPTION
**Pull request by ozkriff**
> [@ozkriff at 2022.01.12](https://github.com/rust-gamedev/rust-gamedev.github.io/issues/888#issuecomment-1010922851): I'll try to prepare the final draft later today <...>

image

Preview (https://github.com/rust-gamedev/rust-gamedev.github.io/pull/909/commits/afaec0f01f00f668294a0458e2dc34fa92fcb7cc): https://rust-gamedev-preview.netlify.app/news/029/

_Part of #888_

### CODE REVIEW

File: content/news/029/index.md.rej [hunk 1]
```diff
@@ -0,0 +1,26 @@
+diff a/content/news/029/index.md b/content/news/029/index.md	(rejected hunks)
```
Comment (17cupsofcoffee) [inline content/news/029/index.md.rej:1]: This looks like it was committed by accident (failed local merge?)

File: content/news/029/index.md [hunk 2]
```diff
@@ -260,22 +257,104 @@ and a thombstone in the game!
...
 [bitgun-name]: https://loglog.games/pages/name-your-zombie/
 
+### [vange-rs WebAssembly Port][vangers-wasm-post]
+
+![Cars, some terrain with roads and lava](vangers.jpeg)
+
+[vange.rs] is the project of re-implementing the [Vangers] game (from 1998)
+in Rust using modern development practices, parallel computations, and GPU.
+
+This month [@caiiiycuk] ported it on "wasm32-unknown-emscripten" target
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:267]:
```suggestion
This month [@caiiiycuk] ported it to the "wasm32-unknown-emscripten" target
```

File: content/news/029/index.md [hunk 3]
```diff
@@ -260,22 +257,104 @@ and a thombstone in the game!
...
+via wgpu's GLES3 backend and [posted an article][vangers-wasm-post]
+about the process and observations.
+
+The wev version of vange-rs can be [played online here][vangers-wasm-play].
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:271]:
```suggestion
The web version of vange-rs can be [played online here][vangers-wasm-play].
```

File: content/news/029/index.md [hunk 4]
```diff
@@ -284,17 +363,105 @@ _Discussions:
...
+[pyxel-examples2]: https://github.com/kitao/pyxel/wiki/Pyxel-User-Examples
+
 ## Learning Material Updates
 
+### [Rust SFML Windows Setup][sfml-video]
+
+[![youtube preview: ping pong window and some dev windows in background](youtube-sfml-win.jpg)][sfml-video]
+
+[@TheFern2] published a [video tutorial][sfml-video] on how to
+set up Rust with [SFML bindings][sfml] on Windows users (using MSVC C++).
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:399]:
```suggestion
set up Rust with [SFML bindings][sfml] for Windows users (using MSVC C++).
```

File: content/news/029/index.md [hunk 5]
```diff
@@ -284,17 +363,105 @@ _Discussions:
...
+
+sfml's [wiki page with instructions for Wndows][sfml-wiki-win] was also updated.
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:401]:
```suggestion
rust-sfml's [wiki page with instructions for Windows][sfml-wiki-win] was also updated.
```

File: content/news/029/index.md [hunk 6]
```diff
@@ -494,46 +682,110 @@ QOI format can fit nicely as a replacement for PNG
...
 
-<!-- Up to 10 most important notes + a link to the full details -->
+## Other News
 
-[See all meeting issues][label_meeting] including full text notes
-or [join the next meeting][join].
+<!-- One-liners for plan items that haven't got their own sections. -->
 
-[label_meeting]: https://github.com/rust-gamedev/wg/issues?q=label%3Ameeting
+- Other game updates
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:709]: Nested list here was a good idea :+1: Might look nicer with a `:` after each subheading?

File: content/news/029/index.md [hunk 7]
```diff
@@ -494,46 +682,110 @@ QOI format can fit nicely as a replacement for PNG
...
+  - [BENDYWORM] now [runs on Linux natively](
+    https://twitter.com/bauxitedev/status/1467817606111498240).
+  - [Rust Shooter progress report][rust-shooter]:
+    enemies are propper player entities with rudimentry AI now,
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:713]:
```suggestion
    enemies are proper player entities with rudimentry AI now,
```

File: content/news/029/index.md [hunk 8]
```diff
@@ -494,46 +682,110 @@ QOI format can fit nicely as a replacement for PNG
...
+    new indoor environment.
+  - [aous] is a GameOff'21 game about the survival of a mutating ant colony.
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:715]:
```suggestion
  - [aous] is a Game Off '21 game about the survival of a mutating ant colony.
```

**Review by 17cupsofcoffee — APPROVED** [review]
A few minor comments, but looks good!

**Comment by kvark** [comment]
is there a preview link?

**Comment by ozkriff** [comment]
Preview: https://rust-gamedev-preview.netlify.app/news/029

**Comment by AngelOnFira** [comment]
It looks like Fish Fight's image isn't referenced properly. I think that's my fault, after converting to jpg, I forgot to change the filetype in the MD file.

**Review by AngelOnFira — APPROVED** [review]

**Comment by ozkriff** [comment]
Merging in ~30m

**Comment by kvark** [comment]
Sorry, was busy and wasn't able to review yet. Will follow up with issues if I find anything.

**Pull request merged**